client/public/ie.js

```diff
@@ -1,22 +1,25 @@
 function get_browser() {
-  var ua = navigator.userAgent, M = ua.match(/(msie|trident)\s*/i) || [];
+  var ua = navigator.userAgent,
+    M = ua.match(/(msie|trident)\s*/i) || [];
   if (/trident/i.test(M[1])) {
     return { name: 'IE' };
   }
   return {
-    name: M[1]
+    name: M[1],
   };
 }
 var browser = get_browser();
-window.onload = function() {
-  if (browser.name === "MSIE" || browser.name === "IE" ) {
-    var tag = document.createElement("p");
+window.onload = function () {
+  if (browser.name === 'MSIE' || browser.name === 'IE') {
+    var tag = document.createElement('p');
     tag.id = 'ie-browser';
-    tag.textContent="This form does not work on Internet Explorer. Please consider using Google Chrome or Firefox instead.";
+    tag.textContent =
+      'This form does not work on Internet Explorer. Please consider using Google Chrome or Firefox instead.';
```

Comment:
We should get Rachit to re-test this feature in IE FYI - we've tried to apply "nice" modern formatting here in the past and discovered it didn't work, although it wasn't specifically single quotes so we might be fine 😄